index.js

```diff
@@ -21,21 +25,53 @@ const deleteItem = (elem) => {
     const items = document.querySelectorAll('.item')
     const itemsCount = items.length
 
-    let myMap = new Map()
-    myMap.set(0, 'two-items')
-        .set(1, 'three-items')
-
-    for (let item of items) {
-        if (!itemsCount) {
-            document.getElementById("portfolio").style.display = 'none'
-            break
-        } else if (myMap.has(itemsCount % 2)) {
-            item.classList.remove('two-items', 'three-items')
-            item.classList.add(`${myMap.get(itemsCount % 2)}`)
-        }
+    const removeClasses = (element) => {
+        return element.classList.remove('two-items', 'three-items')
     }
-}
 
+    if (!itemsCount) {
+        document.getElementById("portfolio").style.display = 'none'
+        return 0
+    } else {
+        let myMap = new Map()
+        myMap.set(2, 'two-items')
+            .set(3, 'three-items')
+
+        myMap.forEach((value, key) => {
+            if (itemsCount % key === 0) {
+                items.forEach(item => {
+                    removeClasses(item)
+                    item.classList.add(value)
+                })
+            }
+        })
+    }
```

Comment:
I'd split this function, it does a lot of things

Reply:
Just Updated this function in last commit. Forgot to do it in this commit

Reply:
I was talking about splitting this function into several functions which will have their single responsibilities.

Reply:
@silchencko thanx! I created new [commit](https://github.com/what-the-hub/landing-page/pull/7/commits/03deb832c03532c5bfb228d48ad49cf660b91e67). Hope it looks better now